chapters/data-analysis.tex

```diff
@@ -1,292 +1,488 @@
+de-identification should not affect the usability of the data.
+In fact, most identifying information can be converted into non-identified variables for analysis purposes
+(e.g. GPS coordinates can be translated into distances). 
+However, if sensitive information is strictly needed for analysis,
+all the tasks described in this chapter must be performed in a secure environment.
 
-\section{Cleaning and constructing derivative datasets}
+% Unique ID and data entry corrections ---------------------------------------------
+There are two main cases when the raw data will be modified during data cleaning.
+The first one is when there are duplicated entries in the data.
+Ensuring that observations are uniquely and fully identified\sidenote{\url{https://dimewiki.worldbank.org/wiki/ID\_Variable\_Properties}}
+is possibly the most important step in data cleaning
+(as anyone who ever tried to merge data sets that are not uniquely identified knows).
+Modern survey tools create unique observation identifiers.
+That, however, is not the same as having a unique ID variable for each individual in the sample.
+You want to make sure the data set has a unique ID variable
+that can be cross-referenced with other records, such as the Master Data Set\sidenote{\url{https://dimewiki.worldbank.org/wiki/Master\_Data\_Set}}
+and other rounds of data collection.
+\texttt{ieduplicates} and \texttt{iecompdup}, 
+two Stata commands included in the \texttt{iefieldkit} 
+package,\index{iefieldkit}
+create an automated workflow to identify, correct and document
+occurrences of duplicate entries. 
 
-Once the de-identified dataset is created, data must be \textbf{cleaned}.\sidenote{
-\url{https://dimewiki.worldbank.org/wiki/Data_Cleaning}}
-\index{data cleaning}
-This process does not involve the creation of any new data fields.
-Cleaning can be a long process: this is the phase at which
-you obtain an extensive understanding of the contents and structure
-of the data you collected in the field.
-You should use this time to understand the types of responses collected,
-both within each survey question and across respondents.
-Understanding this structure will make it possible to do analysis.
+Looking for duplicated entries is usually part of data quality monitoring,
+as is the only other reason to change the raw data during cleaning:
+correcting mistakes in data entry.
+During data quality monitoring, you will inevitably encounter data entry mistakes,
+such as typos and inconsistent values.
+If you don't, you are probably not doing a very good job at looking for them.
+These mistakes should be fixed in the cleaned data set,
+and you should keep a careful record of how they were identified,
+and how the correct value was obtained.
 
-There are a number of tasks involved in cleaning data.
+% Data description ------------------------------------------------------------------
+Note that if you are using secondary data, 
+the tasks described above can likely be skipped.
```

Comment:
_tasks described above can likely be skipped_ - I dont think they should ever be skipped. you have to ensure things like unique ID, look for mistakes etc, were addressed by the person that created and published this data. There are many mistakes in secondary data. So we should say something about that. what we can say tho is that you will most likely spend much less time on this with secondary data, but you should always check for this in any data you are about to use.

Reply:
changed in 968ba4c